docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.5'
+services:
+  test:
+    command: bash -c "yarn install && yarn test"
+    build:
+      context: .
+      target: puppeteer
+    working_dir: /code
+    volumes:
+      - ./src:/code/src
```

Comment:
`.dockerignore` only applies to copying files via the `copy` command in a dockerfile, these are volume mounts so they just mount whatever files are there.

the purpose of this crazy list is to *not* mount `node_modules` because the puppeteer deps installed on osx don't work on ubuntu